tox.ini

```diff
@@ -5,6 +5,11 @@ envlist= py37
 passenv =
     USE_PYTEST_INDEXD
     USE_RUNNING_PG
+    PG_INDEXD_HOST
+
+setenv =
+    NO_PROXY=localhost,postgres
```

Comment:
were both required? I thought only the lowercase no_proxy was needed.

Reply:
@a-collet Could you please take a look at this? 
Thank you!

Reply:
Unfortunately there's no standard to the use of these environment variables. Feel free to try with only setting the lowercase values.

Reply:
If there is no standard for this, I will keep both cases in case future changes in GitLab env break the tests.